proto/net.proto

```diff
@@ -106,6 +106,57 @@ message Weights {
     optional Layer ln2_betas = 6;
   }
 
+  message PolicyHead {
+    // Policy head
+    // Extra convolution for AZ-style policy head
+    optional ConvBlock policy1 = 100;
```

Comment:
Why `100` and not just incremental field numbers?  
Usually it's harder to keep track of what numbers can be used (although probably it our case, it won't).  
And also when serialized, such fields take a bit more space (1 extra byte).

Reply:
Updated to follow serially